INTERPRETING.md

```diff
@@ -160,6 +161,12 @@ located at `test/language/import/nested/dep.js`.
 
 Files bearing a name ending in `.json` are intended to be interpreted as JSON.
 
+Implementers should resolve a specifier of `<do not resolve>` with a TypeError
+throw completion.
+
+Implementers should resolve a specifier of `<abstract module source>` with an
+instance of `%AbstractModuleSource%`.
```

Comment:
```suggestion
Implementers should resolve the specifier `<abstract module source>` to a module that provides a valid [Module Source](https://tc39.es/proposal-source-phase-imports/#sec-module-source-objects), such as a [WebAssembly module](https://webassembly.github.io/esm-integration/js-api/index.html#webassembly-module-record).
```

And also explicitly tag those tests with a flag, such as `source-phase-imports, source-phase-imports-source` with a comment that only engines that can provide at least one module type that has a source can run `source-phase-imports-source`.

Reply:
Updated!